modules/apps/portal-search/portal-search-test-util/src/main/java/com/liferay/portal/search/test/util/groupby/BaseGroupByTestCase.java

```diff
@@ -255,15 +252,13 @@ public void testGroupByTermsSizeDefault() throws Exception {
 
```

Comment:
indexDuplicates cannot be included into for loop because we remove an entry after this